scripts/model.js

```diff
@@ -13,14 +13,30 @@ function Project(object) {
 Project.prototype.toHtml = function() {
   var source = $('#template-project').text();
   var template = Handlebars.compile(source);
-  console.log('handlebars template: ',this);
   return template(this);
 };
 
-projectsData.forEach(function(projectObject){
-  projects.push(new Project(projectObject));
-});
 
 projects.forEach(function(project){
   $('#projects').append(project.toHtml());
 });
+
+Project.loadAll = function(rawData) {
+
+  rawData.forEach(function(ele) {
+    Project.all.push(new Project(ele));
+    console.log(Project.all);
+  });
+};
+
+function fetchAll(){
+  if (localStorage.data){
+    Project.loadAll(JSON.parse(localStorage.data));
+    console.log('localStorage exists.');
+  } else {
+  $.getJSON('data/hackerIpsum.json', function(data){
+    localStorage.data('data', JSON.stringify(data));
```

Comment:
This isn't quite right. Also, indentation for this block is off.

If you're going to save the JSON values into `localStorage`, you should check the ETag to make sure you don't miss out on updated data. Otherwise, it's probably a net negative to use `localStorage`.